tests/licensedcode/data/index/similar_names/licenses/proprietary-license.LICENSE

```diff
@@ -0,0 +1,3 @@
+This component is normally licensed under a proprietary license agreement with 
```

Comment:
Again this commit is repetition of https://github.com/nexB/scancode-toolkit/pull/1534/commits/59857308ba7a70f439f4c4b912c0800cb358d463 .